lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_l10n_helper.rb

```diff
@@ -116,6 +117,30 @@ def self.generate_strings_file_from_hash(translations:, output_path:)
           end
           File.write(output_path, builder.to_xml)
         end
+
+        # Downloads the export from GlotPress for a given locale and given filters.
+        #
+        # Also post-process the downloaded content to comment out any potential line with an empty (`""`) translation,
+        # to ensure those would fall back to the English/Base locale at runtime instead. These case should theoretically
+        # never happen as GlotPress should not include empty translations in its exports, but we've seen issues in the
+        # past, so better safe than sorry.
+        #
+        # @param [String] project_url The URL to the GlotPress project to export from, e.g. `"https://translate.wordpress.org/projects/apps/ios/dev"`
+        # @param [String] locale The GlotPress locale code to download strings for.
+        # @param [Hash{Symbol=>String}] filters The hash of filters to apply when exporting from GlotPress.
+        #                               Typical examples include `{ status: 'current' }` (the default) or `{ status: 'review' }`.
+        # @param [String, IO] destination The path or `IO`-like instance, where to write the downloaded file on disk.
+        #
+        def self.download_glotpress_export_file(project_url:, locale:, filters:, destination:)
+          query_params = (filters || {}).transform_keys { |k| "filters[#{k}]" }.merge(format: 'strings')
```

Comment:
Would this work to keep the chain that evaluates `query_params` a "linear"

```suggestion
        def self.download_glotpress_export_file(project_url:, locale:, destination:, filters: {})
          query_params = filters.transform_keys { |k| "filters[#{k}]" }.merge(format: 'strings')
```

Of course, that would mean omitting `filters: nil` when one doesn't want filters, which I'm guessing is intentional in the design because we expect callers to always pass a filter (as shown by the default value of the param in the action above being `{ status: 'current' }`. 

So I think the answer to this question is "yes it would make the implementation neater, but it would result in a worse ergonomic, and we should keep it as it is."

Reply:
Yeah tbh I went back and forth on that one, and at some point I even had `filters: { status: 'current' }` as a default. But then it didn't read as nice, I thought, and ultimately I decided in 4952fba58ec92373f1f4bf947942fd2e74fe822f that it made more sense to have the default value on the action anyway (see commit's message) rather than having to decide here if we wanted `nil` or `{ status: 'current' }` as a default, and only to potentially have a different default (and no single source of truth for a default value) than the one provided as `default_value` on the `ConfigItem` of the action itself.